### DISCUSSION TITLE
Child template description is same

### DISCUSSION THREAD

**Discussion by pl71** [joomla/joomla-cms#43261] [2. Cold Features]
### Is your feature request related to a problem? Please describe.

My child template has the same description as the Cassiopeia parent theme.

### Describe the solution you'd like

I would like something new as a description.

### Additional context

J5.0.3

**Comment by richard67** [comment]
I think you can change the description in the template-details.xml of your child template.

**Comment by pl71** [comment]
> I think you can change the description in the template-details.xml of your child template.

Yes, it is possible, though not so user-friendly. Thank you.
`\templates\cassiopeia_child\templateDetails.xml`

**Comment by dgrammatiko** [comment]
@pl71 it is hard to do this programmatically because the description could be a translatable variable thus it would be hard/inconsistent to do it automatically. As @richard67 mentioned above you can edit the description field in the `templateDetails.xml` file

**Comment by pl71** [comment]
@dgrammatiko It's not so hard to create a new language file with the child template.  Language variable has to be constructed on the fly by creating the child template. Since you are a template creator, these things matter—also different images for all child templates. Anyway, thank you for the advice.

**Comment by rdeutz** [comment]
Moving it to discussion this needs more details to start developing. Not sure if we should add this at all.

**Comment by brianteeman** [comment]
I started to look at this over my morning coffee. It would seem that it should be possible to at least do some of this from the child template creation screen,
![image](https://github.com/joomla/joomla-cms/assets/1296369/ab2a43de-fa1e-4bef-8856-3749423155a3)
